FindMe/View Controllers/FindMeViewController.m

```diff
@@ -22,11 +22,35 @@ - (void)viewDidLoad
 {
     [super viewDidLoad];
     self.mylocation = LocationManager.shared;
-    [self.mylocation beginTracking];
+    self.trackingButton.backgroundColor =[UIColor redColor];
+    self.trackingButton.layer.cornerRadius = 75;
+    [self setUnSelectedStateOfTrackingButton];
 }
 
 - (IBAction)onFindMe:(id)sender
 {
+    if (!self.trackingButton.isSelected) {
```

Comment:
Can have one method to call and pass this value in as the parameter. Eg: ‘decorateTrackingButtonWithState:SelectedState’